ditojanelidze/exadel-test
# Create game feature

### About Feature
This PR contains all the game creation implementation.

### Changes
* New feature was added
* Functionality was covered with tests
* New swagger docs created

### How To Test
visit 'localhost:300/api-docs' to see swagger docs/.

There is two endpoint.
One for listing all choices and another for playing the game.
To test correctness of the functionality, call POST /play request and
see what was chosen by computer and what was the result of the game.
